python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -1284,6 +1325,17 @@ def main(args):
         convert_safety_checker(pipe, args)
         logger.info("Converted safety_checker")
 
+    if args.convert_unet and args.refiner_version is not None:
+        logger.info(f"Converting refiner")
+        del pipe
+        gc.collect()
+        args.model_version = args.refiner_version
```

Comment:
Could we restore the original model_version after the refiner conversion is done 2 lines down? The name override breaks some filename assumptions in the coremlcompiler calls

Reply:
Yep! Just pushed